docs/modules/servers/pages/basic/help.adoc

```diff
@@ -1,5 +1,54 @@
 = James Basic Server &mdash; Where to get help
 :navtitle: Help
 
-(TODO)
+////
 
+This information is based on the email discussion from here:
+  --> https://www.mail-archive.com/server-dev@james.apache.org/msg66420.html
+
+However, I am not satisfied that I have properly captured the community's
+opinion, and there are only a few people who have contributed to the
+thread.
+
+Would be good to have some comments on this content.
+
+It could just be that I don't like the content. But if that is what the
+community thinks, then that is what we should write.
+
+////
+
+== Warning
+
+Please note that if you want to try to use James, you are pretty
+much on your own.
+
+James is intended for use only by the current committers, or perhaps only by
+competent developers only who have the ability to understand the code as it
+exists now and can make changes themselves without any help. What you see
+is what you get. Take it or leave it.
+
+The James Community is made up of people who "scratch their own itch".
+If you have a problem with the code or the documentation, then please
+step up and fix it rather than asking for help.
+
+If you are a user who wants to try to use James, you should seriously
+consider hiring a professional organization to help you. Most likely,
+James is not for you.
+
+You can try filing issues, but there is absolutely no guarantee it will be
+fixed. If you are lucky and a developer has the same problem as you, then
```

Comment:
`You can try filing issues, but there is absolutely no guarantee it will be
fixed.`

This is mostly untrue.

For the record, I did personnaly fixed several JPA+POP3 obscure buggs, reported by users. I do try to provide help, and understand people bugs, and offer fixes.

I feel offended by this page content. There are differences between "You are on your own" and "The software comes with no warranty but some people will eventually find the time to  help you, without SLA"

Also this statement defeats the entire purpose of the basic server.

Reply:
Sorry, it was not my intent to offend. I also do not feel satisfied with the content, but this is how I interpreted what members were saying in [this discussion](https://www.mail-archive.com/server-dev@james.apache.org/msg66420.html).

Perhaps we could redo this discussion somewhere? I think there may have been some misunderstandings.

Reply:
I must admit I stayed far from this raging discussion.

My take would be:

```
The James community is composed of volunteers (individuals and organizations) wich ship the related softwares without warranty of any kind. However some organisation do offer support (link), don't hesitate to contact their marketting team if needed.

That being said, the James community offers help via different channels (gitter, mail). We try to answer most questions quickly, and likely would fix most issues. (I keep a conditional). Be aware that documented and precise bug reports (with log, configuration samples, and clear explanations are likely to get more attention. 
```


```

Reply:
I started a discussion on the mailing list, so I will resolve this comment. Thanks for brining it up.